time time
float32[] yaws
float32[] pitches
float32[] average_yaws
float32[] average_pitches
float32[] relative_yaws
float32[] relative_pitches
